src/components/Asset/AssetContent/index.tsx

```diff
@@ -45,6 +45,13 @@ export default function AssetContent({
     setIsOwner(isOwner)
   }, [accountId, owner, asset])
 
+  useEffect(() => {
```

Comment:
why is this effect here doubled? There is an effect doing the same thing just above it

Reply:
ohh think this duplication appeared after a v4 merge or rebase 😢 , but good point of moving this out going to add in the `AssetProvider` something similar with what we have for `isAssetNetwork`.